h/templates/client/annotation.html

```diff
@@ -55,34 +51,34 @@
 
   <!-- Excerpts -->
   <section class="annotation-quote-list"
-           ng-repeat="target in vm.target() track by $index"
-           ng-if="vm.hasQuotes()">
-    <excerpt collapsed-height="40"
-             inline-controls="true"
-             overflow-hysteresis="20"
-             content-data="selector.exact">
+    ng-repeat="target in vm.target() track by $index"
+    ng-if="vm.hasQuotes()">
+    <excerpt collapsed-height="35"
+      inline-controls="true"
+      overflow-hysteresis="20"
+      content-data="selector.exact">
       <blockquote class="annotation-quote"
-                  ng-bind-html="selector.exact"
-                  ng-repeat="selector in target.selector
-                             | filter : {'type': 'TextQuoteSelector'}
-                             track by $index"></blockquote>
+        ng-bind-html="selector.exact"
+        ng-repeat="selector in target.selector
+          | filter : {'type': 'TextQuoteSelector'}
+          track by $index"></blockquote>
     </excerpt>
   </section>
 
   <!-- / Excerpts -- >
 
   <!-- Body -->
-  <section name="text" class="annotation-body">
+  <section name="text" class="annotation-body annotation-text">
```

Comment:
What's the reason for introducing a second class here, could we just add the rules to the existing `.annotation-body` class?